common/sgx/report.c

```diff
@@ -53,7 +53,8 @@ static oe_result_t _parse_sgx_report_body(
         sizeof(report_body->mrsigner)));
 
     if (report_body->isvprodid > OE_INT8_MAX)
-        goto done;
+        OE_RAISE(OE_QUOTE_VERIFICATION_ERROR);
```

Comment:
Not your code, but I realized this check is wrong ... this should be a `OE_UINT16_MAX` check since the point of this is to break it into 2 bytes below (it assumes there's some value to shift in the upper bits). Also, it's unclear if this check is needed since `report_body->isvprodid` is already typed to be `uint16_t`

Reply:
Removed check.